migration/src/lib.rs

```diff
@@ -16,7 +16,11 @@ mod m20230203_205959_improve_upsert_perf;
 mod m20230224_093722_performance_improvements;
 mod m20230310_162227_add_indexes_to_bg;
 mod m20230317_121944_remove_indexes_for_perf;
+mod m20230510_183736_add_indices_to_assets;
+mod m20230516_185005_add_reindex_to_assets;
+mod m20230525_115717_cl_audit_table;
 mod m20230526_120101_add_owner_delegate_sequence_number;
+mod m20230528_124011_cl_audit_table_index;
```

Comment:
I think these migration files need to be added to the PR.

Reply:
We can do that in a separate PR, I kept all the cl_audit stuff separate. It's also a massive table so I'm skeptical it should go into the "main" version of DAS

Reply:
yep sounds good